src/reducers/entities/integrations.js

```diff
@@ -89,11 +89,16 @@ function outgoingHooks(state = {}, action) {
 function commands(state = {}, action) {
     const nextState = {...state};
     switch (action.type) {
+    case IntegrationTypes.RECEIVED_COMMANDS:
     case IntegrationTypes.RECEIVED_CUSTOM_TEAM_COMMANDS: {
+        const nextCommands = {};
         for (const command of action.data) {
-            nextState[command.id] = command;
+            if (command.id) {
+                const id = command.id;
+                nextCommands[id] = command;
+            }
         }
-        return nextState;
+        return nextCommands;
```

Comment:
@jwilander are you ok with this changes here?

Reply:
I'm actually going to revert this. This is what is causing the current tests to fail and it's not longer needed since we're filtering for team commands at the selector now.